### PR TITLE
feat: implement the nuxt app

### PR DESCRIPTION
**Pull request by vd3v**
Screenshot 2023-08-23 at 18 35 01

### CODE REVIEW

File: mock-server/public/customer-web.html [hunk 1]
```diff
@@ -683,7 +683,7 @@ <h2 data-v-78a4f09e="" data-test-id="filter-option-label" class="filter-option__
   <!--  STORES    -->
   <section class="stores" id="stores-list">
     <div class="store" data-test-id="store"
-         >
+         style="background-image: url('https://res.cloudinary.com/glovoapp/q_30,f_auto,c_fill,dpr_1.0,h_156,w_351,b_transparent/Stores/ayjw5mlgjkzs1ln5bzh3')">
```
Comment (vd3v) [inline mock-server/public/customer-web.html:686]: I removed it by accident when cleaning up the html for JSX compatibility.

File: mock-server/public/css/variables.css [hunk 2]
```diff
@@ -1,296 +1,299 @@
...
-	--material-color-red-700: #d32f2f;
-	--material-color-red-800: #c62828;
-	--material-color-red-900: #b71c1c;
-	--material-color-red-a100: #ff8a80;
-	--material-color-red-a200: #ff5252;
-	--material-color-red-a400: #ff1744;
-	--material-color-red-a700: #d50000;
+    /* app specific */
+    --footer-wave-desktop: url(https://res.cloudinary.com/glovoapp/image/fetch/f_svg,q_auto:low/https://glovoapp.com/images/landing/footer-wave-desktop.svg);
+    --footer-wave-mobile: url(https://res.cloudinary.com/glovoapp/image/fetch/f_svg,q_auto:low/https://glovoapp.com/images/landing/footer-wave-mobile.svg);
```
Comment (vd3v) [inline mock-server/public/css/variables.css:4]: These 2 vars got lost due to JSX cleanup, bringing them back to this file instead.
The rest is formatting noise.

**Review by claudiaballano — COMMENTED** [review]
Great!

Could you also:

1. Remove README or at least add something like 

# Basic Next 3 application
This is a very simple application built with **Nuxt 3**, aimed to do a performance test

# Learn more
To learn more about Next take a look at the following resources:

 2. Remove event loop thing

**Review by claudiaballano — APPROVED** [review]
Giving u the approval to no block the performance test

**Comment by vd3v** [comment]
> Great!
> 
> Could you also: remove README or at least add something like
> 
> ```
> Basic Nuxt 3 application
> This is a very simple application built with **Nuxt 3**, aimed to do a performance test
> 
> Learn more
> To learn more about Next take a look at the following resources:
> ```
> 
> and remove event loop thing, just to clean up a bit more the app

Added it here
https://github.com/paolo-chiabrera/two-of-one/pull/33

**Pull request merged**